### ISSUE TITLE
dex.storage is not a function

### ISSUE THREAD

**Issue by claudebarde** [madfish-solutions/quipuswap-sdk#10]
I followed the examples and I received the following error message: `dex.storage` is not a function. I tried to use the value in `dex.storage` as `dexStorage` but it wouldn't work either.
I solved the problem by using `dex.contract.storage()` instead.

**Comment by serg-plusplus** [comment]
Yes. After V2, the API was changed. The `storage` property inside `dex` is a snapshot of the storage at the time of the `findDex()` called.

**Comment by claudebarde** [comment]
I updated the README file for you with the new piece of information you provided ;)
